src/Layout/Header/SearchBox/index.jsx

```diff
@@ -98,7 +91,13 @@ export function SearchBox () {
   )
 }
 
-const options = [
-  { label: 'dupa' },
-  { label: 'dupa2' }
-]
+function generateLocalizations (amountOfLocalizations) {
```

Comment:
Let's add new endpoint called `localizationOptions` to our `json-server` and fetch it. Using faker in the app is not a good idea, because one day the fake app may become a real app and you will need to find all the faker usage and rewrite it to use the backend enpoints. Who know maybe you will need to write a real backend for this app.